src/main/java/org/jenkinsci/plugins/credentialsbinding/impl/ZipFileBinding.java

```diff
@@ -53,6 +54,7 @@
         secret.unzipFrom(credentials.getContent());
     }
 
+    @Symbol("zip")
```

Comment:
🐛  I think - didn't we already have a zip step somewhere?

Maybe zipCredential or zipSecret?